index.html

```diff
@@ -648,6 +680,45 @@
     return array;
 }
 
+function undoLastPath(){
+    var lastPath = drawnPaths.pop()
+    // nothing in the path history
+    if (!lastPath || !lastPath.points){
+        console.warn('nothing to undo')
+        return
+    }
+    
+    // draw undo arcs over the selected area
+    lastPath.points.forEach(function(point) {
+        ctx.beginPath();
+        ctx.arc(point.x, point.y, lastPath.brushSize, 0, Math.PI * 2);
+        ctx.closePath();
+        ctx.fill();
+        tempCtx.beginPath();
+        tempCtx.arc(point.x, point.y, lastPath.brushSize, 0, Math.PI * 2);
+        tempCtx.closePath();
+        tempCtx.fill();
+    })
+    updateImage({ 
```

Comment:
IIUC: this will essentially do an "undo" brush over the last path if it was a blur or paint (and a blur if it was an undo), correct? If so, I have a couple points:
- If the last path overlaps with a previous path, part of that previous path could end up completely uncovered. I'm not sure I see a better thing to do with that situation, but wanted to highlight it.
- If the last path was an undo, then we'll blur over it -- even if what it was actually undoing was a paint. Since the undo could cover both blur and paints, or could cover something that was not originally blurred, the behavior might be kinda weird.

Reply:
Thanks! See notes [here](https://github.com/everestpipkin/image-scrubber/pull/18#issuecomment-637481462)